doc/api/inspector.md

```diff
@@ -0,0 +1,60 @@
+# Inspector
+
+> Stability: 1 - Experimental
+
+The `inspector` module provides an API for interacting with the V8 inspector.
+
+It can be accessed using:
+
+```js
+const inspector = require('inspector');
```

Comment:
This _might_ conflict with existing [npm "inspector" package](https://www.npmjs.com/package/inspector) by @AndreasMadsen. Pinging him to see if that package is still useful.